### PR TITLE
feat(zapp_login_plugin_oauth_tv_2_0): add logout confirmation screen

### PR DESCRIPTION
**Pull request by alashuk-sc**
## Description: 
Add Logout confirmation screen.

### Ticket: 
https://applicaster.monday.com/boards/1615228456/views/35072856/pulses/2123873478?userId=17198506

### What's done:
1. Added the ability to use the logout confirmation screen with configurable title, description and buttons (for logout and cancel logout)
2. Added configurable values for SIGN IN screen: 
- separator width;
- sign in client logo;
3. Added configurable values for Action buttons:
- width;
- height;
- border radius;

LOGOUT CONFIRMATION SCREEN: 
Screenshot_1642666095

SIGN IN SCREEN: 
Screenshot_1642665990

### CODE REVIEW

**Review by kononenkoAnton — APPROVED** [review]

File: plugins/zapp_login_plugin_oauth_tv_2_0/manifests/localizations.config.js [hunk 1]
```diff
@@ -77,14 +77,19 @@ const common = [
     initial_value: "Sign Out",
   },
   {
-    key: "sing_out_url_text",
-    label: "Sign Out url descriptin",
-    initial_value: "To update your account please visit",
+    key: "cancel_sing_out",
```
Comment (ranmeirman) [inline plugins/zapp_login_plugin_oauth_tv_2_0/manifests/localizations.config.js:80]: typo - sing-sign

File: plugins/zapp_login_plugin_oauth_tv_2_0/manifests/localizations.config.js [hunk 2]
```diff
@@ -77,14 +77,19 @@ const common = [
...
+    label: "Cancel Sign Out button text",
+    initial_value: "Cancel",
   },
   {
-    key: "sing_out_url",
-    label: "Sign Out url",
-    initial_value: "www.google.com",
+    key: "sing_out_title",
```
Comment (ranmeirman) [inline plugins/zapp_login_plugin_oauth_tv_2_0/manifests/localizations.config.js:85]: typo

**Review by ranmeirman — CHANGES_REQUESTED** [review]

File: plugins/zapp_login_plugin_oauth_tv_2_0/manifests/localizations.config.js (hunk 1 above)
Comment (alashuk-sc) [reply plugins/zapp_login_plugin_oauth_tv_2_0/manifests/localizations.config.js]: fixed

File: plugins/zapp_login_plugin_oauth_tv_2_0/manifests/localizations.config.js (hunk 2 above)
Comment (alashuk-sc) [reply plugins/zapp_login_plugin_oauth_tv_2_0/manifests/localizations.config.js]: fixed

**Review by snooky23 — APPROVED** [review]

**Pull request merged**